en/restaurant/create-restaurant.md

```diff
@@ -0,0 +1,128 @@
+title: Create a Restaurant
+lang: en
+ref: creation-restaurant
+parent: Restaurant's guide
+nav_order: 5
+
+# Create a Restaurant
+
+In order to put your restaurant on CoopCycle, you have to meet with the delivery cooperative which as an `Admin` account on CoopCycle. Only administrators can create new restaurants.
+
+In order to create a new restaurant, administrators have to go click on *restaurant* page and click on `create a new restaurant` green button. 
+
+## Step 1 : Add General information
+
+Now you have to add general information about your restaurant. Some of these informations would be possibly editing later with your restaurant account, but other - such as the terms of the contract or options - are only accessible with adminstrator account. 4 tabs have to be complete :
+
+- [General](#General)
+- [Options](#Options)
+- [Fulfillment](#Fulfillment)
+- [Settlement](#Settlement)
+
+### General
+
+On this `General` tab, you can modify general informations about your restaurant :
+
+- Displayed to clients :
+  - Name of the Restaurant on CoopCycle
+  - Type of Business
+  - Adress
+  - Description of your restaurant 
+  - Webd
+  
+- Not displayed to clients :
+  - Website
+  - Telephone number
+  - SIRET number
+  - VAT number
+  - RCS number
+  
+### Options
+
+4 options have to be decided on this tab :
+
+- Restaurant is exclusive which means that it is only available on CoopCycle
+- Restaurant is featured on homepage 
+- Restaurant also accepts quotes (e.g.:catering)
+- Enable Deposit-Refund system
+
+### Fulfillment
+
+#### Schedules
+On this page, you can chose between two types of activity, or chose both :
+- Collect, which means that the customer come to the restaurant to take-away his or her order
+- Delivery, which means that the cooperative delivers the order to the customer
+
+For each activity, you can edit a schedule which determines when customers can receive their orders or come take them at the restaurant. 
+
+For each schedule you have to chose between *As soon as possible* and *time slot* :
+- *As soon as possible* : The customer receives his or her order as soon as possible. On his or her application, the customer during his or her order sees : "in XX minutes"
+- *Time slot* : The customer choses when he or her receives the order. On his or her application, the customer during his or her order sees : "Between XX PM/AM and XX PM/AM"
+
+For each schedule, you have to chose with the administrator if the restaurant-owner has the right to change these schedules. Click on `allow editing` if it is the case.
+
+#### Global options
+
+- Define a delivery perimeter
+- Define, if needed, an additional delay before ordering. For instance, if you put 1 in the `day` field, customers have to order the day before.
+- Define, if needed, a number of shipping choices
+
+### Settlement
```

Comment:
This guide is for restaurant, no need to put this here. Maybe move this to another page dedicated to explaining how calculations are made? Because there has been lots of confused people about this

Reply:
Ok. For you these settings would be better in the admin's guide ? It seems to me that it is also important to give that information to restaurants.